website/docs/wallet/withdraw-validator.md

```diff
@@ -16,87 +18,68 @@ import TabItem from '@theme/TabItem';
-
-When withdrawing your validator, there are several important concepts to be wary of, namely:
-
-### Validators must be fully exited to trigger a full withdrawal
-
-Full validator withdrawals require your validator to exit first, as exits do not happen automatically. You will need to submit a voluntary exit by following our documentation [here](exiting-a-validator.md). Once your validator exits, it will no longer need to perform its responsibilities after some time (there can be a delay if the validator is part of a sync committee or recently slashed) . The ordering of requests for setting withdrawal credentials or exiting does not matter, once a validator has both its withdrawal credentials updated as well as in an exited state funds will automatically be added to the chosen execution address when processed.
-
-### Full withdrawals are not immediate
-
-Making a **full-validator withdrawal** means that you must exit your validator first. Validator exits have a queue that can only process a few items every few minutes, so your exit can take days or weeks depending on how big this queue grows. The reason there is an exit queue is for the security of the network. Having bounds on validator churn is important towards guaranteeing safety properties of the chain and keeping consensus stable. There is also a queue for activating validators in addition to exiting them. Withdrawals, full or partial, are processed at a rate of at most 16 validators per block.
-
-### Full withdrawals are irreversible
-
-Validator exits are irreversible, and so are full validator withdrawals. Once you exit, you cannot do anything with your validator except for withdraw it. Once you perform a full withdrawal, you will receive your validator balance at the Ethereum address of your choosing, and you cannot revert this action. You could use your withdrawn funds to spin up another separate validator if you wish, however.
-
-### Partial withdrawals do not exit your validator
-
-Partial withdrawals only withdraw your validator earnings to an Ethereum address of your choosing, but do not exit nor require an exit from your validator. That is, if you are running a validator that has a balance of 33.3 ETH, you can perform a bls-to-execution-change request to initiate the partial withdrawal of 1.3 ETH and automatic withdrawals of earnings over 32ETH in the future. Your validator will continue to operate normally
-
-### Partial withdrawals of your earnings will continue indefinitely to your Ethereum address of your choice
+- **Pool:** Upon submission of a validator exit request or bls-to-execution-change request, the message will sit in a special place in memory ( the pool ) to be broadcasted across your peers. Since only the block proposers can include these requests and there is a limit to the number of requests included per block, sometimes if the pool becomes too full your message may be dropped and not included. If this happens, a re-submission of the request may be required.
```

Comment:
We can work on tightening these definitions up over time and possibly replacing the term list with quicklooks, which would allow us to remove this section and instead allow readers to hover over terms for definitions when needed.

Reply:
yeah possibly in the future migrating this to the glossery and mapping against it when referenced in places? just throwing ideas out there

Reply:
Exactly :)